broker/service/api/__init__.py

```diff
@@ -39,10 +39,19 @@
             raise Exception("plugin '%s' section missing" % plugin)
     
     if 'kubejobs' in plugins:
-        k8s_conf_path = config.get('kubejobs', 'k8s_conf_path')
-        count_queue = config.get('kubejobs', 'count_queue')
-        redis_ip = config.get('kubejobs', 'redis_ip')
 
+        # Setting default values for the necessary variables
+        k8s_conf_path = "./data/conf"
```

Comment:
can you define this hardcoded path in a constant at the top of this file? makes it easier for anyone touching this code in the future.

Reply:
Modification made in commit 540b521953feea4e73758f6d8173a1c58e9e65be